diff --git a/node_modules/@jupyterlab/coreutils/lib/url.js b/node_modules/@jupyterlab/coreutils/lib/url.js
index c3d8fef..bbacf62 100644
--- a/node_modules/@jupyterlab/coreutils/lib/url.js
+++ b/node_modules/@jupyterlab/coreutils/lib/url.js
@@ -55,6 +55,7 @@ var URLExt;
         const u = url_parse_1.default(parts[0], {});
         const prefix = `${u.protocol}${u.slashes ? '//' : ''}${u.auth}${u.auth ? '@' : ''}${u.host}`;
         // If there was a prefix, then the first path must start at the root.
+        path_1.posix = path_1;
         const path = path_1.posix.join(`${!!prefix && u.pathname[0] !== '/' ? '/' : ''}${u.pathname}`, ...parts.slice(1));
         return `${prefix}${path === '.' ? '' : path}`;
     }
