diff --git a/node_modules/@types/react-syntax-highlighter/index.d.ts b/node_modules/@types/react-syntax-highlighter/index.d.ts
index 49590f0..92a6e76 100644
--- a/node_modules/@types/react-syntax-highlighter/index.d.ts
+++ b/node_modules/@types/react-syntax-highlighter/index.d.ts
@@ -688,11 +688,21 @@ declare module "react-syntax-highlighter/dist/esm/styles/hljs/vs2015" {
     export default style;
 }
 
+declare module 'react-syntax-highlighter/dist/esm/styles/hljs/vs2015.js' {
+    const style: { [key: string]: React.CSSProperties };
+    export default style;
+}
+
 declare module "react-syntax-highlighter/dist/esm/styles/hljs/xcode" {
     const style: { [key: string]: React.CSSProperties };
     export default style;
 }
 
+declare module 'react-syntax-highlighter/dist/esm/styles/hljs/xcode.js' {
+    const style: { [key: string]: React.CSSProperties };
+    export default style;
+}
+
 declare module "react-syntax-highlighter/dist/esm/styles/hljs/xt256" {
     const style: { [key: string]: React.CSSProperties };
     export default style;
